### ISSUE TITLE
HANA DB Install - License Key before install

### ISSUE THREAD

**Issue by kallass** [sap-linuxlab/community.sap_install#384]
I am trying to automate the license apply also for the HANa installation. But we can only generate permanent license from SAP Support site only if we provide the hardware key from the Database properties in HANA studio.

So how can we supply the license file even before the install. Is there any way to get the HW key before install.

sap_hana_install_apply_license: no
#sap_hana_install_license_path:
#sap_hana_install_license_file_name:

**Comment by Wabri** [comment]
I dont think that is possible to obtain the hardware key before the installation of the hana database. Maybe we can extract the hardware key, than generate the license key and apply to the hana system.
